### PR TITLE
fix: do not display warning if no override

### PR DESCRIPTION
**Pull request by fkm**

### CODE REVIEW

File: addon/helpers/get-widget.js [hunk 1]
```diff
@@ -25,6 +25,11 @@ export default Helper.extend({
   compute([field], { default: defaultWidget = "cf-field/input" }) {
     try {
       const widget = field.question.meta.widgetOverride;
+
+      if (!widget) {
+        return defaultWidget;
+      }
```
Comment (anehx) [inline addon/helpers/get-widget.js]: Would be better if you used `warn("bla bla", widget, { id: "bla-bla" })`

See [here](http://api.emberjs.com/ember/3.9/functions/@ember%2Fdebug/warn)

Comment (fkm) [reply addon/helpers/get-widget.js]: Hm?

Comment (anehx) [reply addon/helpers/get-widget.js]: Gotta remove those lines as well tho

Comment (fkm) [reply addon/helpers/get-widget.js]: :joy:

Comment (fkm) [reply addon/helpers/get-widget.js]: Done

**Pull request merged**